src/web_app/src/pages/ga_hyperParams.tsx

```diff
@@ -87,20 +95,22 @@ export function GaHyperParamsPage() {
                 "momentum"     : 0.2
```

Comment:
why is there a momentum and a learning rate in the GA hyper parameters ?

Reply:
They will not be used. As you said, I send you all and according to the method type you select the corresponding hyperparameters.